docs/powershell/cost/Get-FinOpsCostExport.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+grand_parent: PowerShell
+parent: Cost
+title: Get-FinOpsCostExport
+nav_order: 1
+description: 'Get a list of Cost Management exports.'
+permalink: /powershell/cost/Get-FinOpsCostExport
+---
+
+<span class="fs-9 d-block mb-4">Get-FinOpsCostExport</span>
+Get a list of Cost Management exports.
+{: .fs-6 .fw-300 }
+
+[Syntax](#-syntax){: .btn .btn-primary .fs-5 .mb-4 .mb-md-0 .mr-4 }
+[Examples](#-examples){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [🧮 Syntax](#-syntax)
+- [📥 Parameters](#-parameters)
+- [🌟 Examples](#-examples)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+The Get-FinOpsCostExport command gets a list of Cost Management exports for a given scope.
+
+<br>
+
+## 🧮 Syntax
+
+```powershell
+Deploy-FinOpsHub `
+    [-Name <string>] `
+    [-Scope <string>] `
+    [-DataSet <string>] `
+    [-StorageAccountId <string>] `
+    [-StorageContainer <string>] `
+    [-ApiVersion <string>] `
+```
+
+<br>
+
+## 📥 Parameters
+
+| Name          | Description                                                                                                                                                                          |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Name          | Optional. Name of the export. Supports wildcards.                                                                                                                                           |
+| Scope | Optional. Resource ID of the scope the export was created for. If empty, defaults to current subscription context.                                                                      |
+| DataSet      | Optional. Dataset to get exports for. Allowed values = "ActualCost", "AmortizedCost", "Usage". Default = null (all exports).                                                                                                                  |
```

Comment:
Let's not talk about "Usage". That doesn't give customers reservations or savings plans.
```suggestion
| DataSet      | Optional. Dataset to get exports for. Allowed values = "ActualCost", "AmortizedCost". Default = null (all exports).                                                                                                                  |
```